lib/puppet/functions/libkv/add_libkv.rb

```diff
@@ -0,0 +1,32 @@
+# Add libkv 'extension' to the catalog instance, if it is not present
+#
+# @author https://github.com/simp/pupmod-simp-libkv/graphs/contributors
+#
+Puppet::Functions.create_function(:'libkv::add_libkv') do
```

Comment:
Since this is namespaced, do we need the double name?

Could it just be `libkv::add`?

It looks like it might be more of a `libkv::load` though.

Reply:
The reason I used 'add', is because attribute accessors were added to the catelog.  That said, the name is not that important to me